src/components/TopMenu.tsx

```diff
@@ -1,87 +1,88 @@
 import { IS_SDL_LIVE } from "../constants"
-import { Link } from "react-router-dom"
 import Modal from "./Modal"
 import NetworkDisplay from "./NetworkDisplay"
 import { RewardsBalancesContext } from "../providers/RewardsBalancesProvider"
+import { ReactComponent as SaddleLogo } from "../assets/icons/logo.svg"
 import SiteSettingsMenu from "./SiteSettingsMenu"
 import TokenClaimModal from "./TokenClaimModal"
 import Web3Status from "./Web3Status"
 import classNames from "classnames"
 import { formatBNToShortString } from "../utils"
-import logo from "../assets/icons/logo.svg"
 import useDetectOutsideClick from "../hooks/useDetectOutsideClick"
 import { useTranslation } from "react-i18next"
 
-interface Props {
-  activeTab: string
-}
+type ActiveTabType = "" | "pools" | "risk"
 
-function TopMenu({ activeTab }: Props): ReactElement {
+function TopMenu(): ReactElement {
   const { t } = useTranslation()
   const [currentModal, setCurrentModal] = useState<string | null>(null)
+  const { pathname } = useLocation()
+  const activeTab = pathname.split("/")[1] as ActiveTabType
 
   return (
-    <header
-      data-testid="topMenuContainer"
-      className="top"
-      style={{ border: "#000, solid 3px" }}
-    >
-      <div className="logoWrapper">
-        <Link to="/">
-          <img className="logo" alt="logo" src={logo} />
-        </Link>
-      </div>
-
-      <ul className="nav">
-        <li>
-          <Link
-            data-testid="swapNavLink"
-            to="/"
-            className={classNames({ active: activeTab === "swap" })}
-          >
-            {t("swap")}
-          </Link>
-        </li>
-        <li>
-          <Link
-            to="/pools"
-            className={classNames({
-              active:
-                activeTab === "pools" ||
-                activeTab === "deposit" ||
-                activeTab === "withdraw",
-            })}
-          >
-            {t("pools")}
-          </Link>
-        </li>
-        <li>
-          <Link
-            to="/risk"
-            className={classNames({ active: activeTab === t("risk") })}
-          >
-            {t("risk")}
-          </Link>
-        </li>
-      </ul>
-      <div className="walletWrapper">
-        <RewardsButton setCurrentModal={setCurrentModal} />
-        <Web3Status />
-        <NetworkDisplayAndSettings />
-        <IconButtonAndSettings />
-      </div>
-      <Modal
-        isOpen={!!currentModal}
-        onClose={(): void => setCurrentModal(null)}
+    <AppBar position="static" elevation={0}>
+      <Toolbar
+        data-testid="topMenuContainer"
+        sx={{ mx: { md: 7 }, mt: { md: 3 } }}
       >
-        {currentModal === "tokenClaim" && <TokenClaimModal />}
-      </Modal>
-    </header>
+        <Hidden mdDown>
+          <Box flex={1} flexBasis="30%">
+            <Link to="/">
+              <SaddleLogo />
+            </Link>
+          </Box>
+        </Hidden>
+
+        <ul className="nav">
+          <li>
```

Comment:
Should we be leveraging mUI Links here @hammeiam ?

Reply:
Let me update this on the another PR with styled of MUI.
MUI link don't works well for menu because it makes page refresh like opening with _blank on the <a/> tag.